### DISCUSSION TITLE
安卓手机使用PAC自动代理或者手动代理，B站app出现无法联网现象

### DISCUSSION THREAD

**Discussion by SHIELD-SKY** [1715173329/UnblockNeteaseMusic#1082] [Q&A] [answered]
### Bug 描述

1. 安卓手机使用PAC自动代理或者手动代理，B站app出现无法联网现象。

2. 另一个问题是，安卓手机目前需要安卓证书吗？我发现网易云莫名其妙的解锁了地域限制，但是没有替换无版权音频。而且，我在其他人的安卓机上测试，发现两种代理都无效，怀疑需要安装证书？

### 预期行为

根据自动生成的PAC，代理不应该影响B站app啊。

### 实际行为

b站app打开显示离线。
但是其他应该正常。

### 日志内容

我用一键脚本安装的
https://shell.xiu2.xyz/#/md/unblock163

日志好像不大好开启。

### 网易云音乐版本号

V8.9.70.212604

### 操作系统

ColorOS V12.1

### 问题排查

- [X] 我确认我使用的核心是由 UnblockNeteaseMusic 项目官方发行，不是其他任何 fork。
- [X] 我确认我已经升级到了最新的核心版本（推荐使用最新构建而不是 release）。
- [x] 我确认我已经启用了 HTTPS 端口。
- [ ] 我确认我已经正确设置了 EndPoint。
- [ ] 我确认我已经在对应的客户端正确安装了 CA 证书。

**Comment by 1715173329** [comment]
> 怀疑需要安装证书？
> [X] 我确认我已经在对应的客户端正确安装了 CA 证书。

请问你打的勾是给谁看的？

> 我用一键脚本安装的

本项目已提供安装脚本 https://github.com/UnblockNeteaseMusic/install-scripts

**Comment by SHIELD-SKY** [comment]
额 不好意思，我以为问题排查必须打勾，才能submit。 
EndPointinturl我开的是
        http://music.163.com
用的一键脚本的默认设置。不知道对不对。

我mac OS客户端配置了CA证书，可以正常使用。
安卓手机前两天没有装CA 也莫名其妙的可以用。
然后在其他人的安卓机上就无效。

**Comment by 1715173329** [comment]
本来就是要求必须安装证书

**Comment by SHIELD-SKY** [comment]
我安装的是证书在User下？难道必须装在System下吗？ 我看手机浏览器显示起作用了。信任了链接

**Comment by SHIELD-SKY** [comment]
另外，打开了PAC代理，B站app确实显示无连接。是不是因为PAC里哪个IP地址影响了b站的app啊

**Comment by 1715173329** [comment, marked as answer]
> 难道必须装在System下吗？

对

> 是不是因为PAC里哪个IP地址影响了b站的app啊

如果当前网络环境，网易云音乐和 bilibili 分配到了同一家 CDN，就会出现问题

**Comment by SHIELD-SKY** [comment]
谢谢您的解答
